File: RxDemo/src/main/java/com/tamsiree/rxdemo/activity/ActivityAutoImageView.java
package com.tamsiree.rxdemo.activity;

import android.os.Bundle;
import android.widget.RelativeLayout;

import com.tamsiree.rxdemo.R;
import com.tamsiree.rxkit.RxBarTool;
import com.tamsiree.rxkit.RxDeviceTool;
import com.tamsiree.rxui.activity.ActivityBase;
import com.tamsiree.rxui.view.RxTitle;

import butterknife.BindView;
import butterknife.ButterKnife;

/**
 * @author tamsiree
 */
public class ActivityAutoImageView extends ActivityBase {

    @BindView(R.id.rx_title)
    RxTitle mRxTitle;
    @BindView(R.id.activity_auto_image_view)
    RelativeLayout mActivityAutoImageView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        RxBarTool.noTitle(this);
        RxBarTool.setTransparentStatusBar(this);
        setContentView(R.layout.activity_auto_image_view);
        ButterKnife.bind(this);
        RxDeviceTool.setPortrait(this);

        mRxTitle.setLeftFinish(mContext);
    }
}

File: RxDemo/src/main/java/com/tamsiree/rxdemo/activity/ActivityAutoImageView.kt
package com.tamsiree.rxdemo.activity

import android.os.Bundle
import com.tamsiree.rxdemo.R
import com.tamsiree.rxkit.RxBarTool.noTitle
import com.tamsiree.rxkit.RxBarTool.setTransparentStatusBar
import com.tamsiree.rxkit.RxDeviceTool.setPortrait
import com.tamsiree.rxui.activity.ActivityBase
import kotlinx.android.synthetic.main.activity_auto_image_view.*

/**
 * @author tamsiree
 */
class ActivityAutoImageView : ActivityBase() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        noTitle(this)
        setTransparentStatusBar(this)
        setContentView(R.layout.activity_auto_image_view)

    }

    override fun initView() {
        setPortrait(this)
        rx_title.setLeftFinish(mContext)
    }

    override fun initData() {

    }
}